### PR TITLE
Handle . in column names

### PR DESCRIPTION
**Pull request by tomtor**
### Describe your changes :

Fix https://github.com/open-metadata/OpenMetadata/issues/981

#
### Type of change :

- [x] Bug fix

afbeelding

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@harshach @ayush-shah @pmbrull

### CODE REVIEW

File: ingestion/src/metadata/ingestion/sink/metadata_rest.py [hunk 1]
```diff
@@ -160,6 +160,11 @@ def write_tables(self, table_and_db: OMetaDatabaseAndTable):
                     table_id=created_table.id, sample_data=table_and_db.table.sampleData
                 )
             if table_and_db.table.tableProfile is not None:
+                for pd in table_and_db.table.tableProfile[0]:  # Only for entry 0?
```
Comment (kad-vijlbc) [inline ingestion/src/metadata/ingestion/sink/metadata_rest.py:163]: Not sure if this is the correct way to handle this

Comment (harshach) [reply ingestion/src/metadata/ingestion/sink/metadata_rest.py:163]: it's better to loop here and fix it. Just in case if we get profiles for multiple days

Comment (tomtor) [reply ingestion/src/metadata/ingestion/sink/metadata_rest.py]: Fixed

**Comment by harshach** [comment]
Thanks @tomtor for the PR

**Pull request merged**